src/index.js

```diff
@@ -0,0 +1,102 @@
+import './style.css';
+// import { format } from 'date-fns';
```

Comment:
Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

Reply:
Fixed